sdkproject/Assets/Mapbox/Unity/Editor/FeatureBehaviourEditor.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEditor;
+using Mapbox.Unity.MeshGeneration.Components;
+
+[CustomEditor(typeof(FeatureBehaviour))]
```

Comment:
For something as simple as this, I recommend using [`ContextMenu` attribute](https://docs.unity3d.com/ScriptReference/ContextMenu.html). It's faster to write, leads to one less class, and keeps the inspector a bit cleaner.